boa/src/object/gcobject.rs

```diff
@@ -992,6 +1010,112 @@ impl GcObject {
         // 7. Return target.
         Ok(())
     }
+
+    pub fn set_integrity_level(
```

Comment:
This should also have documentation.

Reply:
Also, could you move this function to `object::operations`?